manim_slides/ipython/ipython_magic.py

```diff
@@ -125,7 +125,7 @@ def construct(self):
         in a cell and evaluate it. Then, a typical Jupyter notebook cell for Manim Slides
         could look as follows::
 
-            %%manim_slides -v WARNING --progress_bar None MySlide --manim-slides controls=true data_uri=true
+            %%manim_slides -v WARNING --progress_bar None MySlide --manim-slides controls=true one_file
```

Comment:
I am not sure about this one. I am not familiar with IPython magic and was not able to test it. Please, @jeertmans, fix if necessary.

Reply:
Didn't you forget the leading `--`?

Reply:
Also, it should be a hyphen, `--one-file`.

Reply:
Ah, my bad. This is a template config variable, so `one_file=true`.

Reply:
Fixed!